llarp/router/router.cpp

```diff
@@ -266,7 +288,7 @@ namespace llarp
       LogError("RC is invalid, not saving");
       return false;
     }
-    diskworker()->addJob(std::bind(&Router::HandleSaveRC, this));
+    QueueDiskIO(std::bind(&Router::HandleSaveRC, this));
```

Comment:
std::bind is sort of obsolete now because you can do pretty much everything it does with a lambda, and the lambda often generates more efficient code, especially when the size/number of captures is small:

```C++
QueueDiskIO([this] { HandleSaveRC(); });
```
(it's also shorter).